pootle/apps/pootle_fs/forms.py

```diff
@@ -21,18 +21,22 @@
 class ProjectFSAdminForm(forms.Form):
 
     fs_type = forms.ChoiceField(
+        # Translators: Pootle FS backend to deal with translations.
         label=_("Filesystem backend"),
-        help_text=_("Select a backend filesystem"),
+        help_text=_("Select a filesystem backend"),
         choices=(),
         widget=forms.Select(
             attrs={'class': 'js-select2'}))
     fs_url = forms.CharField(
+        # Translators: This tells where to find the translations.
         label=_("Backend URL or path"),
         help_text=_(
             "The URL or path to your translation files"))
     translation_path = forms.CharField(
+        # Translators: Expression reflecting how the filesystem looks like.
+        label=_("Translation path mapping"),
         help_text=_(
-            "The translation path mapping for your filesystem"))
+            "This mapping tells how the translations filesystem looks like"))
```

Comment:
Please don't repeat yourself, This is uneeded, we know the help is about this mapping.

Translation path mapping that maps the localisation files on the filesystem to stores on Pootle.